util/ranger/testdata/ranger_suite_out.json

```diff
@@ -392,8 +392,13 @@
       {
         "SQL": "select * from t;",
         "Plan": [
-          "TableReader_5 4.00 root partition:all data:TableFullScan_4",
-          "└─TableFullScan_4 4.00 cop[tikv] table:t keep order:false"
+          "PartitionUnion_9 10004.00 root  ",
+          "├─TableReader_11 1.00 root  data:TableFullScan_10",
+          "│ └─TableFullScan_10 1.00 cop[tikv] table:t, partition:p0 keep order:false",
+          "├─TableReader_13 3.00 root  data:TableFullScan_12",
+          "│ └─TableFullScan_12 3.00 cop[tikv] table:t, partition:p1 keep order:false",
+          "└─TableReader_15 10000.00 root  data:TableFullScan_14",
+          "  └─TableFullScan_14 10000.00 cop[tikv] table:t, partition:p2 keep order:false, stats:pseudo"
```

Comment:
The partition p2 will never be used, we ignore it in the query

Reply:
I have created an [issue](https://github.com/pingcap/tidb/issues/22619) to describe this.